modules/mono/editor/GodotTools/GodotTools.ProjectEditor/ProjectGenerator.cs

```diff
@@ -12,26 +11,29 @@ public static class ProjectGenerator
     {
         public static string GodotSdkAttrValue => $"Godot.NET.Sdk/{GeneratedGodotNupkgsVersions.GodotNETSdk}";
 
+        private const string ProjectTemplate = "project.csproj.template";
+
         public static ProjectRootElement GenGameProject(string name)
         {
             if (name.Length == 0)
                 throw new ArgumentException("Project name is empty.", nameof(name));
 
-            var root = ProjectRootElement.Create(NewProjectFileOptions.None);
-
-            root.Sdk = GodotSdkAttrValue;
-
-            var mainGroup = root.AddPropertyGroup();
-            mainGroup.AddProperty("TargetFramework", "net6.0");
-            mainGroup.AddProperty("EnableDynamicLoading", "true");
-
             string sanitizedName = IdentifierUtils.SanitizeQualifiedIdentifier(name, allowEmptyIdentifiers: true);
 
-            // If the name is not a valid namespace, manually set RootNamespace to a sanitized one.
+            var assembly = typeof(ProjectGenerator).Assembly;
+            var template = new StreamReader(assembly.GetManifestResourceStream(ProjectTemplate)).ReadToEnd();
```

Comment:
Yeah, we can do that. I prefer templates way more than coding this stuff by hand usually, they're much more readable than trying to follow what the code is doing (and we can have comments in templates, which we can't with the API). But since things are simple again, we can go back to the original code, that's easy enough :+1: